com.Acestream.engine.json

```diff
@@ -0,0 +1,149 @@
+                    "config-opts": ["--disable-glibtest"],
+                    "cleanup": ["/bin"],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://download.gnome.org/sources/pygobject/2.28/pygobject-2.28.7.tar.xz",
+                            "sha256": "bb9d25a3442ca7511385a7c01b057492095c263784ef31231ffe589d83a96a5a"
+                        }
+                    ]
+                }
+            ]
+        },
+            "libs/libappindicator/libappindicator-gtk2-12.10.json",   
+         {
+            "name": "acestream",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm755 apply_extra /app/bin/apply_extra",
+                "install -Dm755 acestream.sh /app/bin/acestream",
+                "for size in 16 24 32 48 64 128 256; do\n                    rsvg-convert -w $size -h $size -f png -o $size.png logo-small.svg\n                    install -Dm644 $size.png /app/share/icons/hicolor/${size}x${size}/apps/com.Acestream.engine.png\n                done",
+                "install -Dm644 logo-small.svg export/share/icons/hicolor/scalable/apps/com.Acestream.engine.svg",
+                "install -Dm644 com.Acestream.engine.appdata.xml /app/share/appdata/com.Acestream.engine.appdata.xml",
+                "install -Dm644 com.Acestream.engine.desktop /app/share/applications/com.Acestream.engine.desktop"
+            ],
+            "sources": [
+                {
+                    "type": "script",
+                    "dest-filename": "apply_extra",
+                    "commands": [
+                        "tar -zxvf acestream.tar.gz",
+                        "rm -f acestream.tar.gz"
+                    ]
+                },
+               {
+                    "type": "script",
+                    "dest-filename": "acestream.sh",
+                    "commands": [
+                        "exec /app/extra/start-engine --client-gtk --log-file $XDG_DATA_HOME/acestream.log \"$@\""
+                    ]
+                },                                                                            
+                {
+                    "type": "extra-data",
+                    "filename": "acestream.tar.gz",
+                    "only-arches": [
+                        "x86_64"
+                    ],
+                    "url": "http://download.acestream.media/linux/acestream_3.1.49_ubuntu_18.04_x86_64.tar.gz",
+                    "sha256": "d2ed7bdc38f6a47c05da730f7f6f600d48385a7455d922a2688f7112202ee19e",
+                    "size": 17001150,                    
+                    "x-checker-data": {
+                        "type": "html",
+                        "url": "http://download.acestream.media/linux/",
+                        "url-pattern": "(http://download.acestream.media/linux/acestream_[0-9.]+_ubuntu_[0-9.]+_x86_64.tar.gz)",
```

Comment:
Does the `ubuntu_18.04` part actually change that often?

Reply:
yes, ubuntu version can change by link